k9mail-library/src/main/java/com/fsck/k9/mail/ServerSettings.java

```diff
@@ -16,10 +16,13 @@
  * @see com.fsck.k9.mail.store.StoreConfig#getTransportUri()
  */
 public class ServerSettings {
+
+    public enum Type { IMAP, SMTP, WebDAV, POP3 }
```

Comment:
Splitting ServerSettings is doable except that the WebDAV server settings provides both incoming and outgoing, which complicates things. Given WebDAV for email is deprecated, can I suggest leaving this until support is removed from K-9? 

Happy to sort out the naming as you suggest.